### PR TITLE
[pt] feat: add localization concepts index portugues

### PR DESCRIPTION
**Pull request by EzzioMoreira**
Add concepts `index` translation for Portuguese.

---

**Preview**: https://deploy-preview-4901--opentelemetry.netlify.app/pt/docs/concepts/

### CODE REVIEW

**Review by joaopgrassi — APPROVED** [review]

File: content/pt/docs/concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Conceitos do OpenTelemetry
+linkTitle: Conceitos
+description: Principais conceitos do OpenTelemetry
+aliases: [concepts/overview]
```
Comment (edsoncelio) [inline content/pt/docs/concepts/_index.md:5]: I think this alias config can be removed.
Could you please confirm that, @chalin?

**Review by lgfa29 — COMMENTED** [review]
✅

File: content/pt/docs/concepts/_index.md (hunk 1 above)
Comment (EzzioMoreira) [reply content/pt/docs/concepts/_index.md:5]: @edsoncelio the docs in JA and ZH have the aliases. I copied and pasted rsrsrsrsrsrs

Comment (chalin) [reply content/pt/docs/concepts/_index.md:5]: Yes, generally speaking, such aliases, which pre-date the creation of the Portuguese translation, can be dropped.

Comment (chalin) [reply content/pt/docs/concepts/_index.md]: FYI, I've removed the `aliases` field.

**Review by chalin — APPROVED** [review]

File: content/pt/docs/concepts/_index.md (hunk 1 above)
Comment (EzzioMoreira) [reply content/pt/docs/concepts/_index.md]: Thanks 👍

**Pull request merged**